phoenix-core-client/src/main/java/org/apache/phoenix/jdbc/ZKConnectionInfo.java

```diff
@@ -83,6 +95,10 @@ public ReadOnlyProps asProps() {
             //This has the highest priority
             connectionProps.put(HConstants.CLIENT_ZOOKEEPER_QUORUM, getZkHosts());
         }
+        String zkHostsWithoutPorts = getZkHostsWithoutPorts();
+        if (zkHostsWithoutPorts != null) {
+            connectionProps.put(HConstants.ZOOKEEPER_QUORUM, zkHostsWithoutPorts);
```

Comment:
@virajjasani We should be keeping this property the same as the  CLIENT_ZOOKEEPER_QUORUM, Right!, 
Since that's the ZK spec and should have the right format/spec to make the connection. The only backward compatibility issue was that it was not set earlier causing upstream application to misbehave.

Reply:
If that's enough, that would certainly make life MUCH easier.

Reply:
The backward compatibility issue is that 5.1 client never returned port number from ConnectionInfo whereas here if we don't do this, 5.2 client will always have port info.

Reply:
Alright, let's keep them same with release note that downstreamer will have to take care of this.